packages/shared/src/contexts/BootProvider.tsx

```diff
@@ -202,9 +202,10 @@ export const BootDataProvider = ({
     async (newUser: LoggedUser | AnonymousUser) => {
       const updated = updateLocalBootData(cachedBootData, { user: newUser });
       setCachedBootData(updated);
-      await queryClient.invalidateQueries(
-        generateQueryKey(RequestKey.Profile, newUser),
-      );
+      await queryClient.invalidateQueries({
+        queryKey: generateQueryKey(RequestKey.Profile, newUser),
+        refetchType: 'all',
```

Comment:
refetch inactive query as well (since profile one will be due to `useProfile` hook not being mounted until redirect to user profile page)